README.md

```diff
@@ -160,25 +160,22 @@ ShopifyCheckoutKit.configuration.backgroundColor = .systemBackground
 
 Initializing a checkout session requires communicating with Shopify servers and, depending on the network weather and the quality of the buyer's connection, can result in undesirable waiting time for the buyer. To help optimize and deliver the best experience, the SDK provides a preloading hint that allows app developers to signal and initialize the checkout session in the background and ahead of time.
 
-Preloading is an advanced feature that can be disabled via a runtime flag:
-
+Preloading is an advanced feature that will only be activated after calling:
 ```swift
-ShopifyCheckoutKit.configure {
-  $0.preloading.enabled = false // defaults to true
-}
+ShopifyCheckoutKit.preload(checkout: checkoutURL)
 ```
 
-Once enabled, preloading a checkout is as simple as:
-
+Once preload is called, the checkout view is cached. You are expected to re-call preload every time you need to refresh the checkout view. If for whatever reason you need to invalidate the cache without preloading/loading the checkout view again (e.g we use this during testing of our sample app), you can call this function:
 ```swift
-ShopifyCheckoutKit.preload(checkout: checkoutURL)
+ShopifyCheckoutKit.configuration.preloading.clearCache()
```

Comment:
I think this would be better? wdyt? (Sorry, I know I suggested `clearCache` 🤦‍♂️ )

```suggestion
ShopifyCheckoutKit.invalidateCache()
```